service-commands/src/commands/service-commands.json

```diff
@@ -28,7 +28,7 @@
   "contracts": {
     "up": [
       "cd contracts/",
-      "docker run --name audius_ganache_cli -d -p 8545:8545 --network=audius_dev trufflesuite/ganache-cli:latest -h 0.0.0.0 -l 8000000 --acctKeys contracts-ganache-accounts.json -a 100 -b 5",
+      "docker run --name audius_ganache_cli -d -p 8545:8545 --network=audius_dev trufflesuite/ganache-cli:v6.9.1 -h 0.0.0.0 -l 8000000 --acctKeys contracts-ganache-accounts.json -a 100 -b 5 --deterministic true",
```

Comment:
are we moving off of latest because `--deterministic` is deprecated? what if we just use `-d`?

Reply:
we're pinning to the same version as eth-contracts uses. contracts/ will be deprecated soon so i think this is the best way of ensuring stability. then we can change when we upgrade to ganache 7 for eth contracts - wdyt

Reply:
i think we should pin ganache until we modify our solidity framework (forge instead of truffle or smth along those lines)

Reply:
sounds good but maybe we should just use `-d` so it's compatible for both.

```
  -d, --wallet.deterministic            Use pre-defined, deterministic seed.
                                        deprecated aliases: --deterministic         [boolean] [default: false]
```

Reply:
oooo good call yes 👍

Reply:
Sounds good - I'll add this in the fast-follow PR after ID is wrapped up